src/autorun.js

```diff
@@ -6,7 +6,7 @@ var child_process = require('child_process');
 var winston = require('winston');
 
 module.exports = {
-    autorun: function(dir) {
+    autorun: function (dir) {
```

Comment:
Why was your beautify different than mine? Aren't we using the same settings? I provided my config.

Reply:
Initially i ran beautify with default settings and the build-failed,  then  only i noticed the config provided and changed my settings and made a commit again. Sorry for the trouble.